.travis.yml

```diff
@@ -1,10 +1,64 @@
 language: php
-         
+
+sudo: required
```

Comment:
Done. You can follow the new build here: https://travis-ci.org/kleisauke/php-vips/builds/178867468